### DISCUSSION TITLE
QNAP QTS

### DISCUSSION THREAD

**Discussion by GalacticPrez** [filecoin-project/lotus#7753] [General Q&A]
I did a search for QNAP and QTS but nothing came up, is there any work done on being able to supply storage to filecoin from a QNAP NAS ? I don't have a huge amount of free storage, but I would like to learn more about these things, and so I figured I could just offer 1TB to store IPFS files with filecoin etc., as a way of learning more about it.

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx

**Comment by LexLuthr** [comment]
@GalacticPrez Can you please explain a bit more about what you are trying to achieve here? Do you want to use NAS to store data? Are you planning to become a storage provider or miner?

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
